### PR TITLE
Docs: Update README and general documentation

### PR DESCRIPTION
**Pull request by Elfo404**
Updates setup instruction and docs.
Moves drone-related things inside the dronereceiver directory. this should make things a bit better organized.

Fixes https://github.com/grafana/grafana-ci-otel-collector/issues/125

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,189 +1,84 @@
...
-Web Interface                 http://127.0.0.1:4040
-Forwarding                    https://3dfc-2001-818-d8d9-a00-e5-c197-b7d2-3551.ngrok-free.app -> http://localhost:8080
-```
+[otel-configuration]: https://opentelemetry.io/docs/collector/configuration/
 
-Copy the forwarding url (in this case `https://3dfc-2001-818-d8d9-a00-e5-c197-b7d2-3551.ngrok-free.app`) and use it to configure the `DRONE_SERVER_PROXY_HOST` environment variable in the `.env` file.
+## Running Tempo, Loki, and Prometheus
 
-### GitHub OAuth App
+A basic docker-compose file is provided in the root of the repository to run Tempo, Loki, and Prometheus.
```
Comment (dsotirakis) [inline README.md:74]: Nit: add link to the file

File: README.md [hunk 2]
```diff
@@ -1,189 +1,84 @@
...
-Click on "Register application".
-
-After the application is registered, generate a `Client secret`.
-
-Take note of the `Client ID` and `Client secret` values and use them to configure the `DRONE_GITHUB_CLIENT_ID` and `DRONE_GITHUB_CLIENT_SECRET` environment variables in the `.env` file.
-
-### Run Drone
-
-You can now start Drone with:
+to start the services, run:
```
Comment (dsotirakis) [inline README.md:76]:
```suggestion
To start the services, run:
```

File: receiver/dronereceiver/README.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+## Local Drone instance
+
+It is possible to use a local Drone instance for easier development.
+
+> [!NOTE]  
+> Note that by default no webhook events are sent to the receiver from GitHub (i.e. when
+> pushing to a branch to trigger a build), so make sure your pipelines can be run manually.
+>
+> If you want to trigger pipelines in your local Drone instance when pushing, you need to setup
+> webhooks in your repository. Refer to Drone docs for more informations.
```
Comment (dsotirakis) [inline receiver/dronereceiver/README.md:61]: Nit: Link?

File: receiver/dronereceiver/README.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+```bash
+docker compose up -d
+```
+
+And use the ngrok forwarding url to access the Drone UI.
+Navigate to the repository you want to start monitoring and click on "Activate repository".
+
+> [!NOTE]  
+> This also starts Tempo, Loki and Prometheus from the `docker-compose.yml` file in the root of the repository. If you only want to start drone you can use
```
Comment (dsotirakis) [inline receiver/dronereceiver/README.md:142]:
```suggestion
> This also starts Tempo, Loki and Prometheus from the `docker-compose.yml` file in the root of the repository. If you only want to start Drone you can use
```

File: receiver/dronereceiver/README.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+>
+> ```bash
+> docker compose up -f docker-compose.drone.yml -d
+> ```
+
+### Get your drone token
+
+If you filled in the `GH_HANDLE` environment variable in the `.env` file, your user has admin privileges. You can get your drone token by navigating to https://SOMETHING.ngrok-free.app/account (replace the url with your ngrok forwarding url) and copy the token.
```
Comment (dsotirakis) [inline receiver/dronereceiver/README.md:150]:
```suggestion
If you filled in the `GH_HANDLE` environment variable in the `.env` file, your user has admin privileges. You can get your Drone token by navigating to https://SOMETHING.ngrok-free.app/account (replace the url with your ngrok forwarding url) and copy the token.
```

File: receiver/dronereceiver/README.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+### Configure the collector
+
+Update the `dronereceiver` receiver in the `config.yaml` file to use the [drone token](#get-your-drone-token) from above:
```
Comment (dsotirakis) [inline receiver/dronereceiver/README.md:154]:
```suggestion
Update the `dronereceiver` receiver in the `config.yaml` file to use the [Drone token](#get-your-drone-token) from above:
```

**Review by dsotirakis — COMMENTED** [review]
only nits. I think it looks way better and cleaner now.
@guicaulada and @nafisat2 it'd be great if you could also have a look and approve yourselves after you have made sure that you can actually follow the instructions and set the whole thing up properly :D

File: receiver/dronereceiver/README.md (hunk 3 above)
Comment (Elfo404) [reply receiver/dronereceiver/README.md]: couldn't find the docs on drone website 🤷 

added instructions in f35a611

**Review by guicaulada — APPROVED** [review]

**Pull request merged**